src/txn/txn.c

```diff
@@ -2422,8 +2422,10 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
     WT_SESSION_IMPL *s;
     char ts_string[WT_TS_INT_STRING_SIZE];
     const char *ckpt_cfg;
+    bool has_stable_timestamp;
```

Comment:
I feel the variable name `has_stable_timestamp` may not be correct here. How about `use_timestamp` as we are triggering the RTS when the use_timestamp is true.

Reply:
> I feel the variable name `has_stable_timestamp` may not be correct here. How about `use_timestamp` as we are triggering the RTS when the use_timestamp is true.

Fixed